# Bazaar merge directive format 2 (Bazaar 0.90)
# revision_id: serkan@gentoo.org-20090322154025-il95by1cherlcq3w
# target_branch: ../gtkhotkey-0.2
# testament_sha1: 57cd0c86a0dc8abd9938a6c3453158860577fdfc
# timestamp: 2009-03-22 17:40:48 +0200
# base_revision_id: mikkel.kamstrup@gmail.com-20081028221117-\
#   90lhle1k6yd06j6f
# 
# Begin patch
=== modified file 'configure.ac'
--- configure.ac	2008-10-27 22:28:44 +0000
+++ configure.ac	2009-03-22 15:40:25 +0000
@@ -45,7 +45,7 @@
 AM_PROG_LIBTOOL
 
 
-PKG_CHECK_MODULES(GTK_HOTKEY, [glib-2.0 >= 2.15.6 gobject-2.0 >= 2.15.6 gio-2.0 >= 2.15.6 gtk+-2.0 >= 2.12])
+PKG_CHECK_MODULES(GTK_HOTKEY, [glib-2.0 >= 2.15.6 gobject-2.0 >= 2.15.6 gio-2.0 >= 2.15.6 gio-unix-2.0 >= 2.15.6 gtk+-2.0 >= 2.12])
 AC_SUBST(GTK_HOTKEY_CFLAGS)
 AC_SUBST(GTK_HOTKEY_LIBS)
 

=== modified file 'src/gtk-hotkey-key-file-registry.c'
--- src/gtk-hotkey-key-file-registry.c	2008-10-27 21:09:19 +0000
+++ src/gtk-hotkey-key-file-registry.c	2009-03-22 15:40:25 +0000
@@ -20,6 +20,7 @@
 #include "gtk-hotkey-utils.h"
 
 #include <gio/gio.h>
+#include <gio/gdesktopappinfo.h>
 
 enum  {
 	GTK_HOTKEY_KEY_FILE_REGISTRY_DUMMY_PROPERTY

# Begin bundle
IyBCYXphYXIgcmV2aXNpb24gYnVuZGxlIHY0CiMKQlpoOTFBWSZTWdt3sNEAAqvfgGAQWW///3/v
//q////wUAWY2e5OzGzhw6yOdt2BkkGiYRoxU/RT9Jsk2Qj1Jpp7VMjbSj1A9EM1DagkpJkNqeia
eqHlD0NTTQNA0AAAAAGgSKCEyNTamSeptpE0PRGmjINDQaaA008o0NBw00wQyGmmRkwgGmgDCaNM
mABA0EkiaNDRPVTwGqntNU/U8FT2lPagmNCMh+pD0gNDRsGEW1mjkegLerb7f8WadJbyYFFMIvsx
Socue4i6RKwvx4ZVuZTba66pZV0FYxsb1dKNX7nS3wIsiY5MgReaX2ijfs8Z6SPrWmE6TNbqyaJ8
IdMZMZ1tdTLMH4iLic2xjqKLucy9DUMqgivIIKZ/uFCwPxHWz0yTR+PntCDnLx7aLeb2/uDIc4sB
jBDoFAHSbBXHqoBDF7yz64s5IpcIRkucQWwgN9RI6Oek7vHJWtNtwMEcKhglfJFxGFSsgkQTGwRg
RGDijQI6hob5IjpIDjy4bSATSOKqiiQjFUWKFRiOM5cQTPhLLvtV2wCgeFVinZMmMoJBALtjKXW5
TaibKa3VK8mFJYInYEZV7bVY5lO80DnmAjIQNxMOBMxk7siPnmRqehFpjrJlNLxJxYUBSIhCFki8
RFjSxEYKsxjIpWSsN0I8OWqWyEsiJ4MH04eTFgExNemw15x1spgK/8meguMHYIs2hEdhNbIzHIeC
M1RAox80DMqFVnqIPPF5LhFAOitaZ4S6koGIiZJnCjb1xEV1Gt2tazcGN5dLPVq+VBeTKbyowlp4
hzkRdiLD/QYOviFN1Pwz5AxGrUU1F72McSPzukU1FqPEi55WQlruHDpTRQMKDryIZiE9dD41VPLi
islmz4QqEUxLwihlI7A0qL4iUiwlJIU3J7a4fmYtGiWh8ax2nvxNIvQbNyGmiN6tFPzFfqLKoK9h
82mjCd3OYBx5ILA2XDVW95MTOLLKqEXU7USmSSMQterwc+7xnLr69OS1EAREPAOC1NO29Uy/st+y
W/OBiRb4hsw8bl1PaOGLCOJ82SmZ6nLmnn9TJxiP+Po472Vy/XuYbbg2YhqhEiDwn35WTyeMtWv6
ab8YzeeCd4BXVY7fcGry5hFM38YXw7LD3exyoeFY0ntUCqi1m1Du+V2hMwWT1gdBsG4R+ixUO+Pe
ZhtEsCCQXA8AdmMNJ3CCFZkGA8sVV9HQ+7VLqJnSXQyGkfy6yw8PN3t7OgipLVXpYokd448h7XQP
ZhnzCHdr4M8kLjObdQjaeJYFjztBjLaDIYcSZhTmVdx407TxvGa8GOQaHBIqIw7NQxi3BcRp4jFt
pAXa88NcTaUHXvTR7BmsKWByRtqBLMfc1+RiQ8RkoFdLI4ssnKHCKBYhQzZWM35nA5EFnTygvUl+
xz2n1G8+DqUweAViJGOprXUAPQkMCCSzk0DE26OSup5Gjq8aYsHWGlE4lAikBiagMzJNkyEUp0zv
IkL3YCsaDDQdANAKwQwQpMP+bILFnPj8d1FPNXpHMQ7TP0ZZlRm7A88ORWZT+h4fK7xi1AtAsI7B
g6xbhff6VHIigXMTkOEevlAQ9H2AweoJ7d/M56jAx294uRYJ6rRfwvO48jLTXTFMU3FJpF8A4Asl
dQBPcagtzrc9HavAKGl0Q32rVwQGWJEN/lMebOlNI5oK9IkxiYGRIaFU0KdWUmblCohiIKac9FhY
64u1xpcsF8k5k5yYTxZRGvo+u8bWnC59l6vDlZEC/zGgmyfakSiwLxMRELpYRSkxsJCg2dw3pzmP
3Qxl/9tVvhCA1E2Yr5WrnOdCWRALjAQ+wN6AXgaQXFP3q8q5LGw1SGMYvxopNNo8rHpi6tI3lgDl
WzkmIsiH0yoCjfI2LgU7KhWXKVh5yfvxULII2COBcb6RmyrhUL2zy4igeZRFysIFH1hSaGOK14jj
NQ/8XckU4UJDbd7DRA==
